go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/operator-framework/operator-lib
 
-go 1.21
+go 1.22.0
```

Comment:
Looks like this wasn't automatically set due to bingo. So perhaps `go 1.22`?

Reply:
Ah good call, I did set this myself

Reply:
Fixed in https://github.com/operator-framework/operator-lib/pull/176/commits/c6b3af11f9eb215131fe782978380c0d01ce33ed

Reply:
Looks like `go mod tidy` resets it. reverted in https://github.com/operator-framework/operator-lib/pull/176/commits/b6409b8033ebec33df0c32c3951bac84132fa61f